<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: white url(D:\\QT/images/main.jpg);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="Height">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: white;</string>
    </property>
    <property name="placeholderText">
     <string>Высота, мм</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Высота шкафчика, мм</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ширина шкафчика, мм</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Width">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>120</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: white;</string>
    </property>
    <property name="placeholderText">
     <string>Ширина, мм</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>100</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Глубина шкафчика, мм</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Depth">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>120</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: white;</string>
    </property>
    <property name="placeholderText">
     <string>Глубина, мм</string>
    </property>
   </widget>
   <widget class="QPushButton" name="OK">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>160</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: white;</string>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QLabel" name="iKitchen_text">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>281</width>
      <height>421</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: white;
vertical-align: top;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_ClassModule">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_ModuleType">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Тип модуля</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_CabinetType">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>170</y>
      <width>591</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="RadioPractick">
       <property name="styleSheet">
        <string notr="true">background: rgba(255, 255, 255, 70);</string>
       </property>
       <property name="text">
        <string>Практик</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="RadioClassicModern">
       <property name="styleSheet">
        <string notr="true">background: rgba(255, 255, 255, 70);</string>
       </property>
       <property name="text">
        <string>Современная 
классика</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="RadioClassic">
       <property name="styleSheet">
        <string notr="true">background: rgba(255, 255, 255, 70);</string>
       </property>
       <property name="text">
        <string>Классика</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="RadioModern">
       <property name="styleSheet">
        <string notr="true">background: rgba(255, 255, 255, 70);</string>
       </property>
       <property name="text">
        <string>Модерн</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="RadioMagnolia">
       <property name="styleSheet">
        <string notr="true">background: rgba(255, 255, 255, 70);</string>
       </property>
       <property name="text">
        <string>Магнолия</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>40</y>
      <width>101</width>
      <height>128</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: url(D:\\QT/images/Practick.png);</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>40</y>
      <width>101</width>
      <height>128</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: url(D:\\QT/images/ClassicModern.png);</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_3">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>40</y>
      <width>101</width>
      <height>128</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: url(D:\\QT/images/Classic.png);</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_4">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>40</y>
      <width>101</width>
      <height>128</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: url(D:\\QT/images/Modern.png);</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_5">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>40</y>
      <width>101</width>
      <height>128</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: url(D:\\QT/images/Magnolia.png);</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_6">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>40</y>
      <width>81</width>
      <height>128</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: url(D:\\QT/images/Radius.png);</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Radius">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>170</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_DSPBase">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>270</y>
      <width>351</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>250</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Материал каркаса</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>300</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Материал фасадов</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_MDFBase">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>320</y>
      <width>351</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>250</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Торцовка каркаса</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_TortsBase">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>270</y>
      <width>351</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_FasadCoverBase">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>320</y>
      <width>351</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>300</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Покрытие фасадов</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_FurnitureBase">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>370</y>
      <width>351</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>350</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Петли фасадов</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_LegsBase">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>420</y>
      <width>351</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>400</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ножки шкафчиков</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_FixBase">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>420</y>
      <width>351</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>400</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Крепеж</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_HDFBase">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>370</y>
      <width>351</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>350</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Материал задней стенки</string>
    </property>
   </widget>
   <widget class="QLabel" name="My_iKitchen">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Моя iKitchen:</string>
    </property>
   </widget>
   <widget class="QLabel" name="iKitchen_Calculate">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>530</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Calculate">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>530</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: white;</string>
    </property>
    <property name="text">
     <string>Расписать стоимость</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Change_Material">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>460</y>
      <width>721</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: white;</string>
    </property>
    <property name="text">
     <string>Внести изменения для фасадов и материалов</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Высота тумбочки, мм</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ширина тумбочки, мм</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>100</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Глубина тумбочки, мм</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>530</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Сумма к оплате:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Конфигурация модуля:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="PaySumm">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>570</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: white;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>570</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Внесенные деньги:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>530</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>грн.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>570</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>грн.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Параметры фасадов (для всей кухни)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>230</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Материалы (для всей кухни)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Параметры модуля</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Pay">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>570</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: white;</string>
    </property>
    <property name="text">
     <string>Оплатить</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
